jdaviz/configs/specviz2d/tests/test_parsers.py

```diff
@@ -40,28 +39,6 @@ def test_2d_parser_jwst(specviz2d_helper):
                                          'Wave 6.24985e+00 um', '')
 
 
-@pytest.mark.xfail(reason="Transpose not implemented for spectrum1d input")

```

Comment:
is it worth it to keep these tests, removing the xfail and using .load without 'transpose' specified, so this coverage is maintained?

Reply:
let me try to remind myself why I removed these (and that affects your point above as well about the one failure) 🤔

Reply:
I reintroduced the test file, but without manual transposing (so updated the expected returned shape) and that seems to pass for me both locally and in CI